Make `PyWavelets` and `scipy` optional installs (#195)
JohannesBuchner/imagehash: README.rst

Installation
============

Based on PIL/Pillow Image, numpy, scipy.fftpack (for pHash) and PyWavelets (for wHash)
Easy installation through `pypi`_::

	pip install imagehash

> scipy and PyWavelets are optional. If you intend on using pHash or wHash, you can install the right dependencies using::
> 
> 	pip install imagehash[phash]
> 	pip install imagehash[whash]
> 

Author: ```suggestion
scipy and PyWavelets are optional. If you don't intend on using them, you can install ImageHash using:

  pip install imagehash --no-dependencies
  pip install numpy pillow

```

Changelog
----------

> * Unreleased: Make PyWavelets and scipy optional installs by @Avasam

Author: ```suggestion
* Unreleased: Make PyWavelets and scipy optional if ImageHash is installed with `--no-dependencies` installs by @Avasam
```
